docs/upgrade-guide/upgrade211-30.md

```diff
@@ -0,0 +1,142 @@
+Upgrade guide ACA V3.0 application compitable with ADF V5.0
+
+---
+
+Title: Upgrading from ACA v2.11 to v3.0
+
+---
+
+# Upgrading from ACA v2.11 to v3.0
+
+This guide explains how to upgrade your ACA v2.11 app to work with ADF v5.0.
+Do not skip this task, if you want your application to be updated to a most recent version of ADF(v5.0). Upgrades of multiple versions of ACA cannot be done in one step only, but should follow the chain of sequential updates.
+Note: the steps described below might involve making changes to your code. If you are working with a versioning system then you should commit any changes you are currently working on. If you aren't using versioning then be sure to make a backup copy of your project before going ahead with the upgrade.
+You can't run ng update to update Angular applications more than one major version at a time.
+
+Library updates:
+
+Upgrade ACA app built with ADF v4.11 to a one supporting ADF v5.0, you need to upgrade angular version v14. You need to upgrade angular versions from v10 to v14 step by step(single major version at a time). In terms to support ADF v5.0 you have to follow the below commands.
+
+
+**Upgrade to v11:**
+
+To update angular version from 10.0 to 11.0 by running the below update commands
+
+    • ng update @angular/core@11 @angular/cli@11 –force
+    
+    • npm i
+      
+      
+Below are the some updates when you upgrade to v11.0
+
+    • Angular now requires TypeScript 4.0. ng update will migrate you automatically.
+    
+    • Support for IE9, IE10, and IE mobile has been removed.
+
+
+
+**Upgrade to v12:**
+
+To update angular version from 11.0 to 12.0 by running the below update commands
```

Comment:
Please capitalise Angular everywhere in the guide

Reply:
updated.